website/client/components/inventory/equipment.vue

```diff
@@ -134,25 +136,25 @@ export default {
       searchText: null,
       searchTextThrottled: null,
       costume: false,
-      groupBy: 'type', // or 'class' TODO move to router?
+      groupBy: 'type', // or 'class'
       gearTypesToStrings: Object.freeze({ // TODO use content.itemList?
-        headAccessory: 'headAccessoryCapitalized',
-        head: 'headgearCapitalized',
-        eyewear: 'eyewear',
-        weapon: 'weaponCapitalized',
-        shield: 'offhandCapitalized',
-        armor: 'armorCapitalized',
-        body: 'body',
-        back: 'back',
+        headAccessory: i18n.t('headAccessoryCapitalized'),
```

Comment:
none this.$t -> i18n.t, the problem is that this is defined before the component is setup so this.$t is not yet available